crew install vim does not work

Freshly installed Chromebrew (though devmode is on (obviously) and I've run a couple of crouton chroots) and continued to install vim as per instructions on the homepage. Does not work.

I've tried multiple times to "install vim". Something seems off with the gcc install? I can't even compile a simple Hello World.

```chronos@localhost ~ $ cc test.c
/usr/local/bin/ld: cannot find /usr/local/lib64/libc.so.6
/usr/local/bin/ld: cannot find /usr/local/lib64/ld-linux-x86-64.so.2
collect2: error: ld returned 1 exit status```

The actual "install vim" error:

```chronos@localhost ~ $ crew install vim
vim: Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient.
http://www.vim.org/
version 8.1.0648
The following packages also need to be installed: 
vim_runtime 
Do you agree? [Y/n] 
Proceeding...
vim_runtime: Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient. (shared runtime)
http://www.vim.org/
version 8.1.0648
No precompiled binary available for your platform, downloading source...
                  -=O=-                           #        #        #        #                                          
Archive downloaded
Unpacking archive, this may take awhile...
gcc (GCC) 7.3.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
checking whether make sets $(MAKE)... yes
checking for gcc... gcc
checking whether the C compiler works... no
configure: error: in `/usr/local/tmp/crew/v8.1.0648.tar.gz.dir/vim-8.1.0648/src':
configure: error: C compiler cannot create executables
See `config.log' for more details
vim_runtime failed to install: `./configure --prefix=/usr/local --localstatedir=/usr/local/var/lib/vim --with-features=huge --with-compiledby='Chromebrew' --with-x=no --disable-gui --enable-multibyte --enable-cscope --enable-fontset --enable-perlinterp=dynamic --enable-pythoninterp=dynamic --enable-python3interp=dynamic --enable-rubyinterp=dynamic --disable-selinux` exited with 77```

--- a/README.md
+++ b/README.md
@@ -38,6 +38,11 @@ Please be aware of the fact that developer mode is insecure if not properly conf
 
 Installation
 ------------
+
+The beta, dev, and Canary channels are ***not*** supported and should ***not*** be used with Chromebrew.
+Failure to take notice of this will cause major issues with your Chromebrew installation.
+See issue [#2890](https://github.com/skycocker/chromebrew/issues/2890) and the [FAQ](https://github.com/skycocker/chromebrew/wiki/FAQ) for more details.
+
 Open the terminal with Ctrl+Alt+T and type `shell`.
 
 If this command returns `ERROR: unknown command: shell`, please have a second look at the prerequisites and make sure your Chromebook is in developer mode.

